include/manager.hpp

```diff
@@ -219,6 +219,18 @@ class Manager
      * @throw std::runtime_error
      */
     void SetTimerToDetectVpdCollectionStatus();
+
+    /**
+     * @brief API to register callback for "assetTag" property change.
+     */
+    void registerAssetChangeCallback();
+
+    /**
+     * @brief Callback API to be triggered on "assetTag"property change.
```

Comment:
nit : space after "assetTag" and property

Reply:
"AssetTag" and not "assetTag"

Reply:
agreed